### PR TITLE
Upgrade runner ubuntu version (16 -> 20)

### PR DESCRIPTION
**Pull request by roxanebellot**
# Problem

I used your template and my actions failed because of this. 
I saw your pipeline was failing too, maybe this will fix it !

# Solution

Ask the jekyll runner to use ubuntu 20 and not ubuntu 16. As far as I undestood, since ubuntu 16 is no longer maintened, there is no more available runner matching our criterion.

### CODE REVIEW

**Comment by MichaelCurrin** [comment]
Ah thanks for the heads up.

I usually use the `-latest`

What do you think about that instead?

https://analytics-hbs.vip-develop.twoyou.co/siphonophore-test/

File: .github/workflows/jekyll.yml [hunk 1]
```diff
@@ -9,7 +9,7 @@ on:
 
 jobs:
   build-deploy:
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-20.04
```
Comment (MichaelCurrin) [inline .github/workflows/jekyll.yml]:
```suggestion
    runs-on: ubuntu-latest
```

Comment (roxanebellot) [reply .github/workflows/jekyll.yml]: :ok_hand:

**Review by MichaelCurrin — APPROVED** [review]

**Comment by roxanebellot** [comment]
Hey sorry to bother you, do you think you could add a `hacktoberfest-accepted` label to this PR please ?

**Pull request merged**